features/step_definitions/basic_step.rb

```diff
@@ -0,0 +1,37 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+When(/^I visit "([^"]*)" page$/) do |page|
```

Comment:
I would call this `page_name` instead of `page` - @j0sv we saw what effect this might have within the step definition when we need to use the capybara's `page` object